README.md

```diff
@@ -28,3 +28,4 @@ In [release-it](https://github.com/release-it/release-it) config:
 | addUnreleased  | `false`        | It leaves "Unreleased" title row if set to `true`.                                                                                               |
 | keepUnreleased | `false`        | It leaves "Unreleased" title row unchanged if set to `true`.                                                                                     |
 | addVersionUrl  | `false`        | Links the version to the according changeset.                                                                                                    |
+| head  | `'HEAD'`        | The git revision the new version tag is compared to in the Unreleased URL.                                                                               |
```

Comment:
If you don't mind, could you please wrap `CHANGELOG.md` into single quotes as `'CHANGELOG.md'`. The same way as you made for `'HEAD'` . Thanks!

Reply:
Sure, already done ;-)